integration/test/app/basic/simple_test.go

```diff
@@ -69,34 +61,56 @@ func TestAppLifecycle(t *testing.T) {
 		Namespace: namespace,
 	}
 
-	// Test creation.
 	{
-		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating chart %#q", customResourceReleaseName))
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating chart value for release: %#q", customResourceReleaseName))
 
-		chartValues, err := chartvalues.NewAPIExtensionsAppE2E(sampleChart)
+		chartValues, err = chartvalues.NewAPIExtensionsAppE2E(sampleChart)
 		if err != nil {
 			t.Fatalf("expected %#v got %#v", nil, err)
 		}
 
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("created chart value for release: %#q", customResourceReleaseName))
```

Comment:
Let'S drop the colons everywhere. 

```suggestion
		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("created chart value for release %#q", customResourceReleaseName))
```